### PR TITLE
[TT-XLA][PyTorch] Training Workload for Qwen 3 8B

### LINKED ISSUE #412: LoRA finetuning: Qwen3-8B
## Description
Add LoRA finetuning support for the Qwen3-8B model in tt-blacksmith.

## Goal
Successfully run a complete LoRA finetuning workflow for Qwen3-8B through tt-blacksmith, validating the model's compatibility with the training pipeline.

## Acceptance Criteria
- [ ] Model loads correctly in tt-blacksmith
- [ ] LoRA adapters can be attached and trained
- [ ] Finetuning run completes without errors
- [ ] Resulting model produces valid outputs

### PR DESCRIPTION
**Pull request by abogdanovicTT**
**Issue**
Closes #412

**Training Workload Setting**

Model: Qwen 3 8B
Strategy: LoRA
Framework: PyTorch
Dataset: Alpaca
Hardware: WH QuietBox

**What's Changed**
This PR introduces training of Qwen 3 8B on WH QuietBox.

**Experiment Results**

<table>
   <tr>
    <td><div align="center">Train Loss</div></td>
    <td><div align="center">Val Loss</div></td>
  </tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
</table>

**Checklist**

- [x]  Addition of appropriate .yaml file
- [x]  TT device testing
- [ ]  GPU testing

### CODE REVIEW

**Review by agobeljicTT — APPROVED** [review]
LGTM
If you wish, you can run a quad-gpu just as a reference check, but it's not necessary.

**Review by mmilosevicTT — APPROVED** [review]
LGTM, please update PR title
